src/git-source-provider.ts

```diff
@@ -177,6 +177,7 @@ export async function getSource(settings: IGitSourceSettings): Promise<void> {
     } else {
       fetchOptions.fetchDepth = settings.fetchDepth
       fetchOptions.fetchTags = settings.fetchTags
+      fetchOptions.showProgress = settings.showProgress
```

Comment:
In v3 before this option was added it was applied to `fetch-depth: 0` usage as well. Can you add this outside the if block, like on like 162?